program_166/program_166.js

```diff
@@ -0,0 +1,10 @@
+let hands = ["rock", "paper", "scissor"]
+
+// Create a function that returns a random item from the array
+function getRandomItem() {
+    let randomNumber = Math.floor(Math.random() * 3)
+    // return randomNumber for debug
```

Comment:
What exactly you are dubbiging here?

Reply:
check if get code for get random index correct

Reply:
Yeah, it is. I just don't understand this comment (just delete it)